publish-cli/src/main/java/com/ericsson/eiffel/remrem/publish/cli/CliOptions.java

```diff
@@ -80,6 +80,7 @@ public static void createCLIOptions() {
         options.addOption("mp", "messaging_protocol", true, "name of messaging protocol to be used, e.g. eiffel3, eiffelsemantics, default is eiffelsemantics");
         options.addOption("domain", "domainId", true, "identifies the domain that produces the event");
         options.addOption("cc", "channelsCount", true, "Number of channels connected to message bus, default is 1");
+        options.addOption("wcto", "wait_for_confirms_timeOut", true, "the timeout for wait for confirms, default is 5 sec");
```

Comment:
Or, rather "default is 5000 ms/milliseconds", because unit is milliseconds, not seconds, isn't it?

Reply:
same here